.github/workflows/caches.yml

```diff
@@ -0,0 +1,79 @@
+name: Reusable CI dependency container images caching workflow
+
+on:
+  workflow_call:
+    inputs:
+      config-path:
+        required: false
+        type: string
+
+# Actions Used (please keep this documented here as added)
+#  https://github.com/marketplace/actions/cache
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+      # Localstack
+      - name: Restore Localstack Image Cache if it exists
+        id: cache-docker-localstack
+        uses: actions/cache@v3
+        with:
+          path: ci/cache/docker/localstack
+          key: cache-docker-localstack-2.1
```

Comment:
can you drag this "2.1" somehow up into a const or env variable.. its just that it then is used in about 10 different spots in the next 10 lines - easy for someone to miss changing it when bumping the version..